pkg/events/groups.go

```diff
@@ -77,3 +77,23 @@ func (GroupMemberRemoved) Unmarshal(v []byte) (interface{}, error) {
 	err := json.Unmarshal(v, &e)
 	return e, err
 }
+
+// GroupFeature represents a group feature
+type GroupFeature struct {
+	Name  string
+	Value string
+}
+
+// GroupFeatureChanged is emitted when a group feature was changed
+type GroupFeatureChanged struct {
+	Executant *user.UserId
+	UserID    string
+	Features  []GroupFeature
```

Comment:
We don't need a timestamp?

Reply:
Good question, I used the `UserFeatureChanged` as a template, and it also does not have a timestamp field.

Reply:
Oki. Then it is consistent at least :smile: